//#ifndef FINDPWD_H
//#define FINDPWD_H

//#include <QMainWindow>
//#include "changepwd.h"

//namespace Ui {
//class FindPwd;
//}

//class FindPwd : public QMainWindow
//{
//    Q_OBJECT

//public:
//    explicit FindPwd(QWidget *parent = nullptr);
//    ~FindPwd();
//    QRect pass_cpGeo;

//signals:
//    void toLogin();
//    void sendUid(const QString &uid);
//    void passToLogin();

//public slots:
//    void receiveGeo(const QRect &geo);

//private:
//    Ui::FindPwd *ui;
//    ChangePwd *cp;
//};

//#endif // FINDPWD_H
